src/test/test_mp_pcg.c

```diff
@@ -24,9 +24,13 @@
 #include "HYPRE_parcsr_mv.h"
 
 #include "HYPRE_IJ_mv.h"
+#include "HYPRE_IJ_mv_mup.h"
```

Comment:
This shouldn't be here.  It is supposed to be included automatically by way of HYPRE_IJ_mv.h if you have HYPRE_MIXED_PRECISION defined and do *not* define hypre_MP_BUILD (see the bottom of the header file).  The same comment holds for the other '*_mup.h' includes here.  The user compile should have no additional mixed precision variables defined except for the first one.

Reply:
@victorapm you can 'ignore' this file if you want, as I am working on cleaning up the tests.